src/AddIns/Uno.UI.Svg/Uno.UI.Svg.csproj

```diff
@@ -0,0 +1,70 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras" ToolsVersion="15.0">
+
+	<PropertyGroup>
+		<TargetFrameworks>xamarinmac20;MonoAndroid12.0;xamarinios10;net461;netstandard2.0</TargetFrameworks>
+		<TargetFrameworksCI>xamarinmac20;MonoAndroid11.0;MonoAndroid12.0;xamarinios10;net461;netstandard2.0</TargetFrameworksCI>
+		<NoWarn>$(NoWarn);NU1701</NoWarn>
+		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+		<Deterministic>true</Deterministic>
+		<Nullable>enable</Nullable>
+		<LangVersion>10.0</LangVersion>
+	</PropertyGroup>
+
+	<PropertyGroup Condition="'$(UnoTargetFrameworkOverride)'!=''">
+		<TargetFrameworks>$(UnoTargetFrameworkOverride)</TargetFrameworks>
+	</PropertyGroup>
+
+	<ItemGroup>
+	  <None Remove="buildTransitive\Uno.UI.Svg.targets" />
+	</ItemGroup>
+
+	<ItemGroup>
+	  <Content Include="buildTransitive\Uno.UI.Svg.targets">
+	    <PackagePath>build</PackagePath>
+	    <Pack>true</Pack>
+	  </Content>
+	</ItemGroup>
+	<ItemGroup>
+		<PackageReference Include="Uno.SourceGenerationTasks" />
+
+		<PackageReference Include="Svg.Skia" />
+		<PackageReference Include="SkiaSharp.Views.Uno" Condition="'$(UNO_UWP_BUILD)'=='true'" />
+		<PackageReference Include="SkiaSharp.Views.Uno.WinUI" Condition="'$(UNO_UWP_BUILD)'!='true'" />
+	</ItemGroup>
+
+	<ItemGroup Condition=" $(IsMonoAndroid) or $(IsXamarinIOS) or $(IsXamarinMac) ">
+		<Reference Include="System.Numerics" />
+		<Reference Include="System.Numerics.Vectors" />
+	</ItemGroup>
+
+	<Import Project="..\..\Common.targets" />
+	<Import Project="..\..\Uno.CrossTargetting.props" />
+
+	<ItemGroup>
+	  <ProjectReference Include="..\..\Uno.Foundation\Uno.Foundation.csproj" />
+	  <ProjectReference Include="..\..\Uno.UI\Uno.UI.csproj" />
+	</ItemGroup>
+
+	<Import Project="..\..\SourceGenerators\Uno.UI.SourceGenerators\Content\Uno.UI.SourceGenerators.props" />
```

Comment:
Same question, which source generator(s) do we need for this project?

Note: If none, consider deleting this line and also the Uno.SourceGenerationTasks package reference.

Reply:
I think we need the native constructor generators for the `SvgCanvas` control, unless I am mistaken